elasticsearch/src/main/java/org/hibernate/search/elasticsearch/cfg/ElasticsearchEnvironment.java

```diff
@@ -49,11 +49,38 @@
 	 * {@code hibernate.search.default.elasticsearch.host=http://myeshost.com:9200}), i.e. only a single Elasticsearch
 	 * cluster is supported for all the indexed entities. This limitation will be removed in a future version of
 	 * Hibernate Search.
-	 * <p>
 	 */
 	public static final String SERVER_URI = "elasticsearch.host";
 
 	/**
+	 * Property for specifying the host names and HTTP ports of the Elasticsearch servers to connect to.
+	 * <p>
+	 * A string is expected.
+	 * <p>
+	 * Defaults to no username (anonymous access).
+	 * <p>
+	 * Can only be given <b>globally</b> (e.g.
+	 * {@code hibernate.search.default.elasticsearch.username=ironman}), because. only a single Elasticsearch
```

Comment:
I'm confused about the "e.g." usage here. You say it has to be global, but then you make an example which uses an index-specific property.

(I understand the reasons, just pointing out the explanation is oddly phrased)